supabase/migrations/20260826094120_block_r2_direct_manifest_jsonb_writes.sql

```
@@ -0,0 +1,247 @@
+    FROM pg_catalog.unnest(p_manifest) AS entry(file_name, s3_path, file_hash)
+    WHERE NOT EXISTS (
+      SELECT 1
+      FROM public.manifest AS m
+      WHERE m.app_version_id = p_version_id
+        AND m.s3_path = entry.s3_path
+        AND m.file_hash = entry.file_hash
+    )
+  );
+$$;
+
+ALTER FUNCTION public.app_version_manifest_jsonb_unmigrated(bigint, public.manifest_entry[]) OWNER TO postgres;
```

SQLFluff (4.3.0)

[error] Line is too long (111 > 80).

(LT05)